platform/mk2/libs/STM32F4xx_DSP_StdPeriph_Lib_V1.0.1/Libraries/STM32F4xx_StdPeriph_Driver/inc/stm32f4xx_dma.h

```diff
@@ -356,8 +356,8 @@ typedef struct {
 /** @defgroup DMA_flags_definition
   * @{
   */
-#define DMA_FLAG_FEIF0                    ((uint32_t)0x10800001)
-#define DMA_FLAG_DMEIF0                   ((uint32_t)0x10800004)
+#define DMA_FLAG_FEIF0                    ((uint32_t)0x10000001)
+#define DMA_FLAG_DMEIF0                   ((uint32_t)0x10000004)

```

Comment:
Yep.  The 8 in those first 2 lines seems to be a bug since they mask it out later.  It was causing the assert to fire when I had it enabled.